book/content/credit/credit.md

```diff
@@ -113,7 +113,7 @@ You can use your ORCID iD whenever you’re prompted to do so, give your [truste
     - Deposit it
   - Software
     - Deposit it (github isn't good enough)
-    - Software journals (e.g. [JORS][], [JOSS][])
+    - Software journals (like [JORS][], [JOSS][])
```

Comment:
Are JORS and JOSS supposed to be links?

Reply:
I think the original author intended to work on it in future as this part is commented out. I will add their links though.